package-lock.json

```diff
@@ -6746,6 +6746,18 @@
 				"@babel/runtime": "^7.8.3"
 			}
 		},
+		"@wordpress/e2e-test-utils": {
```

Comment:
I think this is the first time I have ever seen a reasonable package.lock change